src/App.css

```diff
@@ -1,5 +1,68 @@
 .App {
-    width: 60vw;
-    margin: 0 auto;
+	padding: 0 40px;
 }
 
+.App__input {
+	display: flex;
+	align-items: center;
+}
+
+.button-19 {
```

Comment:
I like the look of these buttons. I'm not sure what all of this styling does. I see some effects on mobile (for example background changes) that aren't present on desktop. Nice look overall!